code/modules/client/loadout/loadout.dm

```diff
@@ -43,14 +45,16 @@ GLOBAL_LIST_EMPTY(gear_datums)
 	return 1
 
 /datum/gear
-	var/display_name       //Name/index. Must be unique.
-	var/description        //Description of this gear. If left blank will default to the description of the pathed item.
-	var/path               //Path to item.
-	var/cost = INFINITY           //Number of metacoins
-	var/slot               //Slot to equip to.
-	var/list/allowed_roles //Roles that can spawn with this item.
-	var/list/species_blacklist //Stop certain species from receiving this gear
-	var/list/species_whitelist //Only allow certain species to receive this gear
+	var/display_name               //Name/index. Must be unique.
```

Comment:
why the hell does git think this is a new line?

Reply:
I messed with the spacing of the comments.